src/admin/server.ts

```diff
@@ -28,7 +28,6 @@ export function getServer(contextFactory: ContextFactory): ApolloServer {
     schema: buildSubgraphSchema([
       { typeDefs: typeDefsAdmin, resolvers: resolversAdmin },
     ]),
-    formatError: errorHandler,
```

Comment:
i always wondered what this was doing - now we know. this function is _very_ strict (clearly). the intent makes sense - hiding sensitive information from clients - but i'm not sure about the implementation.

anyway, nice find and fix!